pages/index.js

```diff
@@ -4,6 +4,7 @@ import styled from "styled-components";
 import SearchForm from "@/components/SearchForm";
 import { useState } from "react";
 import Link from "next/link";
+import { linkStyle } from "./movies/[id]";
```

Comment:
if you use a style in multiple places, please move the styled component in a separate file. you can directly export the styled component, no need to create another component for that.